src/phoenix/server/app.py

```diff
@@ -26,22 +28,60 @@
 
 logger = logging.getLogger(__name__)
 
+templates = Jinja2Templates(directory=SERVER_DIR / "templates")
+
+
+@dataclass(frozen=True)
+class Config:
```

Comment:
fyi. `typing.NamedTuple` is also fine, if we are not adding methods to it
```suggestion
class Config(NamedTuple):
```